packages/cf.js/package.json

```diff
@@ -12,12 +12,12 @@
   "license": "MIT",
   "private": true,
   "scripts": {
-    "clean": "rm -rf .rpt2_cache dist",
+    "clean": "git clean -Xdf",
```

Comment:
I don't think we need a thin alias for `git clean`. Better to just remove the `clean` command

Reply:
We have multiple layers of gitignore files and so `git clean` doesn't work the same way at the root level which is why we "dispatch" it to the packages

Reply:
Actually nevermind i just hadn't tried it .. https://stackoverflow.com/questions/9730486/can-you-have-additional-gitignore-per-directory-within-a-single-repo